index.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+const NR = require('node-resque');
+const asCallback = require('ascallback');
+const connectionDetails = {
+    pkg: 'ioredis',
+    host: process.env.REDIS_HOST,
+    password: process.env.REDIS_PASSWORD,
+    port: process.env.REDIS_PORT,
+    database: 0
+};
+
+/**
+ * Construct an executor and call executor.start with the buildConfig
+ * @method execute
+ * @param  {Object}     config                      config passed in from executor-queue
+ * @param  {Object}     config.executor             config for the executor
+ * @param  {String}     config.executor.name        executor name
+ * @param  {Object}     config.executor.options     options to pass into executor's constructor
+ * @param  {Object}     config.buildConfig          buildConfig needed for executor.start
+ */
+function execute(config) {
+    return new Promise((resolve) => {
+        if (config.executor.name === 'queue') {
```

Comment:
I recall at standup that we wanted it to be a single executor. Why are we bothering with this case?